x-pack/plugins/aiops/README.md

```diff
@@ -14,6 +14,16 @@ The route handler sets up `response_stream_factory:responseStreamFactory()` to c
 
 The response stream factory acts as a wrapper to set up the stream itself, the stream state (for example to set if it's running etc.), some custom actions on the stream as well as analysis handlers that fetch data from ES and pass it on to the stream.
 
-## Development
-
-See the [kibana contributing guide](https://github.com/elastic/kibana/blob/main/CONTRIBUTING.md) for instructions setting up your development environment.
+### Analysis details
+
+Here are some more on the steps involved to do Log Rate Analysis:
```

Comment:
You need to put `details` back in here - 

```
Here are some more details on the steps involved to do Log Rate Analysis:
```

Reply:
🤦 Fixed in [01052fa](https://github.com/elastic/kibana/pull/180258/commits/01052fa184a3e560a0f6b1399ae3e658336d9a6f)!